spec.js

```diff
@@ -0,0 +1,37 @@
+
+//spec.js
+browser.ignoreSynchronization = true;
+
+//Open Google.Com page
+describe('To open google browser and search the text "Thought For The Day"', function() {
+  it('To click the link Thought For The Day', function() {
+    browser.driver.get('http://google.com/');
+
+   
+//Maximize the Window   
+    browser.driver.manage().window().maximize();
```

Comment:
Understood is maximizing the browser a required operation for running the tests?